docs/how-to/run-services-reliably.md

```diff
@@ -0,0 +1,74 @@
+# How to run services reliably
+
+Microservice architectures offer flexibility, but they can introduce reliability challenges such as network interruptions, resource exhaustion, problems with dependent services, cascading failures, and deployment issues. Health checks can address these issues by monitoring resource usage, checking the availability of dependencies, catching problems of new deployments, and preventing downtime by redirecting traffic away from failing services.
+
+To help you manage services more reliably, Pebble provides a comprehensive health check feature.
+
+## Use health checks of the HTTP type
+
+A health check of the HTTP type issues HTTP `GET` requests to the health check URL at a user-specified interval.
+
+The health check is considered successful if the check returns an HTTP 200 response. After getting a certain number of failures in a row, the health check is considered "down" (or unhealthy).
+
+### Configure HTTP-type health checks
+
+For example, we can configure a health check of HTTP type named `svc1-up` that checks the endpoint `http://127.0.0.1:5000/health`:
+
+```yaml
+checks:
+  svc1-up:
+    override: replace
+    period: 10s
+    timeout: 3s
+    threshold: 3
+    http:
+      url: http://127.0.0.1:5000/health
+```
+
+The configuration above contains three key options that we can tweak for each health check:
+
+- `period`: How often to run the check (defaults to 10 seconds).
+- `timeout`: If the check hasn't responded before the timeout (defaults to 3 seconds), consider the check an error.
```

Comment:
I think this reads a bit oddly with the parenthesized default in the middle:
```suggestion
- `timeout`: If the check hasn't responded before the timeout, consider the check an error (defaults to 3 seconds).
```

Reply:
Or we could just drop these, if we add the `# default 10s` style comments in the YAML above.

Reply:
A comment in the first example is probably more than enough. I will add comments and drop this list.